package.json

```diff
@@ -1,55 +1,66 @@
 {
   "name": "docpress-base",
   "description": "Base theme for Docpress.",
+  "homepage": "http://docpress/docpress-base#readme",
   "version": "0.6.10",
-  "author": "Rico Sta. Cruz <rico@ricostacruz.com>",
+  "main": "index.js",
+  "author": {
+    "email": "rico@ricostacruz.com",
+    "name": "Rico Sta. Cruz"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/docpress/docpress-base.git"
+  },
   "bugs": {
     "url": "https://github.com/docpress/docpress-base/issues"
   },
   "dependencies": {
-    "autoprefixer": "6.4.0",
-    "browserify": "13.1.0",
-    "debounce": "1.0.0",
-    "dom101": "1.3.0",
-    "github-markdown-css": "2.4.0",
-    "iconfonts": "0.9.0",
-    "normalize.css": "4.2.0",
-    "nprogress": "0.2.0",
-    "onmount": "1.3.0",
-    "pjax": "0.2.4",
-    "postcss": "5.1.2",
-    "pug": "^2.0.0-beta6",
-    "stylus": "0.54.5",
-    "uglifyify": "3.0.2",
-    "ware": "1.3.0"
+    "autoprefixer": "~6.4.0",
+    "browserify": "~13.1.0",
+    "debounce": "~1.0.0",
+    "dom101": "~1.3.0",
+    "github-markdown-css": "~2.4.0",
+    "iconfonts": "~0.9.0",
+    "normalize.css": "~4.2.0",
+    "nprogress": "~0.2.0",
+    "onmount": "~1.3.0",
+    "pjax": "~0.2.4",
+    "postcss": "~5.1.2",
+    "pug": "~2.0.0-beta6",
```

Comment:
I prefer them to be pinned to certain versions rather than in version ranges. This way, we're a bit more ensured that a given docpress version will keep working in the future. There are things like, say:

- `pjax` breaking even with patch versions
- `github-markdown-css` looking different even with a minor version bump

...and I'd rather avoid that.